doc/user_guide/api.rst

```diff
@@ -164,6 +170,95 @@ API Functions
    vconcat
    when
 
+.. _api-theme:
+
+Theme
+-----
+.. currentmodule:: altair.theme
+
+.. autosummary::
+   :toctree: generated/theme/
+   :nosignatures:
+
+   enable
+   get
+   names
+   register
+   themes
+   unregister
+   ThemeConfig
+   AreaConfigKwds
+   AutoSizeParamsKwds
```

Comment:
# Preview
I think this order makes the most sense.

- Functions will be the first thing you need
- The top level config is clearly visible
- Then further config via `...Kwds`.

![image](https://github.com/user-attachments/assets/49b6963d-96b5-419a-b7a5-6296ea44a82d)